seed/tests/data/vcr_cassettes/geocode_base_case.yaml

```diff
@@ -2,69 +2,113 @@ interactions:
 - request:
     body: null
     headers:
-      Accept: ['*/*']
-      Accept-Encoding: ['gzip, deflate']
-      Connection: [keep-alive]
-      User-Agent: [python-requests/2.20.0]
+      Accept:
```

Comment:
Any reason why this file changes so much?

Reply:
I'm not sure why the format changed. 

I did expect a few of these to change because the date/time, etc of the request is captured, and I did change some addresses behind the requests (effectively completely changing the request &response)

Reply:
okay. doesn't seem like this will impact anything. so move along...